.eslintrc.json

```diff
@@ -11,9 +11,12 @@
   "overrides": [
     {
       "files": ["./src/typeDefs/**/*.ts"],
-      "processor": "@graphql-eslint/graphql"
+      // "processor": "@graphql-eslint/graphql"
```

Comment:
Remove this comment.

Reply:
@EshaanAgg Have done a comment  it not to produce new errors but the newly upgraded version of typescript-eslint-plugin is getting conflict to override to lint graphql code in typeDefs/.ts files and producing this error instead:

![image](https://github.com/PalisadoesFoundation/talawa-api/assets/117595548/59b79344-3a92-41e8-a441-8a0f5ceead92)

Reply:
and if  we use     
 ```
 "parser": "@graphql-eslint/eslint-plugin",
    "plugins": ["@graphql-eslint"]
```
instead

`      "processor": "@graphql-eslint/graphql"`

getting  error of typescript code import . Advising to move the graphql  code to .graphql files